frontend/public/module/k8s/index.ts

```diff
@@ -247,15 +247,22 @@ export type PodSpec = {
   serviceAccountName?: string;
   priorityClassName?: string;
   tolerations?: Toleration[];
+  nodeName?: string;
+  hostname?: string;
   [key: string]: any;
 };
 
 type PodPhase = 'Pending' | 'Running' | 'Succeeded' | 'Failed' | 'Unknown';
 
+type ContainerStateValue = {
+  reason?: string;
+  [key: string]: any;
+};
+
 export type ContainerState = {
```

Comment:
We should probably use https://github.com/openshift/console/blob/master/frontend/public/module/k8s/index.ts#L255-L259

(and update that type if needed)